scipy/stats/_stats_py.py

```diff
@@ -854,6 +867,9 @@ def tstd(a, limits=None, inclusive=(True, True), axis=0, ddof=1):
     return np.sqrt(tvar(a, limits, inclusive, axis, ddof))
 
 
+@_axis_nan_policy_factory(
+    lambda x: x, n_outputs=1, result_to_tuple=lambda x: (x,)
+)
 def tsem(a, limits=None, inclusive=(True, True), axis=0, ddof=1):
```

Comment:
`axis` is currently ignored. Looks like it would be pretty easy to make it work.

Reply:
Done, thanks for noticing that!